odo/__init__.py

```diff
@@ -1,73 +1,37 @@
-from __future__ import absolute_import, division, print_function
+from __future__ import absolute_import as _
+from __future__ import division as _
+from __future__ import print_function as _
+
+from pkg_resources import DistributionNotFound as _DistributionNotFound
+from pkg_resources import get_distribution as _get_distribution
+
+try:
+    __version__ = _get_distribution('odo').version
```

Comment:
we use versioneer to manage this variable, this should be restored to what it was before.